src/dc/util.js

```diff
@@ -10,3 +10,8 @@ export function hypotenuse(a, b) {
   }
   return 0;
 }
+
+export function abs(a) {
+  if (a >= 0) return a;
+  return -a;
```

Comment:
Please use `0 - a`. It leads to usually more performant code because it cannot generate the special `-0` value.